xml/art_suseconnect_tracking.xml

```diff
@@ -0,0 +1,405 @@
+Do you want to delete these systems? (y/n) y
+Purged systems that have not contacted this RMT since 2021-06-16.
+</screen>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="system-data-gathered-by-scc">
+  <title>System data gathered by SCC</title>
+
+  <para>
+   When a system is registered directly via SCC, or its registration information is forwarded by RMT
+   or &suma;, SCC collects the following information:
+  </para>
+
+  <table>
+   <title>System attributes collected by SCC</title>
+   <tgroup cols='3' align='left'>
+    <colspec colnum="1" colname="1" colwidth=""/>
+    <colspec colnum="2" colname="2" colwidth=""/>
+    <colspec colnum="3" colname="3" colwidth="3*"/>
+    <thead>
+     <row>
+      <entry>
+       System Attribute
+      </entry>
+      <entry>
+       Type
+      </entry>
+      <entry>
+       Example value
+      </entry>
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>
+        Hostname
+      </entry>
+      <entry>
+        string
+      </entry>
+      <entry>
+       virtual.domain.net
+      </entry>
+     </row>
+     <row>
+      <entry>
+        CPUs
+      </entry>
+      <entry>
+       int
+      </entry>
+      <entry>
+       1
+      </entry>
+     </row>
+     <row>
+      <entry>
+       Sockets
+      </entry>
+      <entry>
+       int
+      </entry>
+      <entry>
+       2
+      </entry>
+     </row>
+     <row>
+      <entry>
+       Architecture
+      </entry>
+      <entry>
+       string
+      </entry>
+      <entry>
+       x86_64
+      </entry>
+     </row>
+     <row>
+      <entry>
+       UUID
+      </entry>
+      <entry>
+       uuid
+      </entry>
+      <entry>
+       6A5072A0-311B-430E-8EDE-A8770788B92D
+      </entry>
+     </row>
+     <row>
+      <entry>
+       Hypervisor
+      </entry>
+      <entry>
+       string
+      </entry>
+      <entry>
+       KVM
+      </entry>
+     </row>
+     <row>
+      <entry>
+       Cloud Provider
+      </entry>
+      <entry>
+       string
+      </entry>
+      <entry>
+       Amazon (can also be 'Google' or 'Azure', not 100% accurate in the 'Azure' case)
+      </entry>
+     </row>
+     <row>
+      <entry>
+       Last Seen Date
+      </entry>
+      <entry>
+       date
+      </entry>
+      <entry>
+       2021-05-05 (the last time that the system contacted &scc;)
+      </entry>
+     </row>
+    </tbody>
+   </tgroup>
+  </table>
+ </sect1>
+ <sect1 xml:id="suseconnect-benefits">
+  <title>Benefits</title>
+
+  <para>
+   The new functionality of &suseconnect; bring the following benefits to the customer:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     Ability to identify all types of deployments of systems, no matter if they are derived from a
+     custom image or clones of an already registered virtual machine.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     By contacting SCC or RMT regularly, you can obtain the number of actually running registered
```

Comment:
How come sometimes the code says `SCC`, while others `&scc;`?

Reply:
I've used the long version 'SUSE Customer Center' only initially to introduce it, then just a shortcut similar to RMT